dist/backdraft.js

```diff
@@ -1476,6 +1544,70 @@ _.extend(Plugin.factory, {
       this.trigger("draw", arguments);
     },
 
+    exportData : function(sUrl) {
+      var oSettings = this.dataTable.fnSettings;
+      var aoData = this.dataTable._fnAjaxParameters( oSettings );
+      this._addServerParams( aoData );
+      this._fetchCSV( sUrl, aoData );
+    },
+
+    _fetchCSV : function (sUrl, aoData) {
+      if (this.serverSideFiltering) {
+        var filterJson = {};
+        filterJson.name = "ext_filter_json";
+        filterJson.value = this._getFilteringSettings();
+        aoData.push(filterJson);
+      }
+      $.ajax({
+        url: sUrl,
+        data : aoData,
+        dataType : "text",
+        cache : false,
+        type : this.ajaxMethod || "GET",
+        success: function(response, status, xhr) {
```

Comment:
I'm pretty sure we _can't_ do that due to how we're passing parameters for how to make the table (in a post request).  Setting window.location is equivalent to doing a get request at a URL, which doesn't work for getting table data within the reporting infrastructure we're creating with DataTables/Backdraft.

If I could have just set window.location, I would have done that.